deploy/manifests/custom-metrics-resource-reader-cluster-role.yaml

```diff
@@ -6,9 +6,10 @@ rules:
 - apiGroups:
   - ""
   resources:
-  - namespaces
   - pods
-  - services
```

Comment:
why do we need to remove namespaces and services?

Reply:
this setup is the one that is used in the helm charts. please see here:
https://github.com/helm/charts/blob/master/stable/prometheus-adapter/templates/resource-metrics-cluster-role.yaml

in addition I've had this bug #274, and after setting the cluster role as suggested in this pull request everything worked perfectly.

no need for namespaces and services.